src/crowdin_gui.h

```diff
@@ -83,7 +83,7 @@ class CrowdinSyncDestination : public CloudSyncDestination
 {
 public:
     wxString GetName() const override { return "Crowdin"; }
-
+    bool Auth(wxWindow* parent) override;
```

Comment:
I don't understand why this is done. It is committed as part of "Crashed on open cached Crowdin file if logged out", but none of it seems to be related to crashing anything, and surely crashes can be avoided in a simpler way?

There's also a commit that seems related with a message "Move AttachCloudSync() to GUI code" which *does no such thing at all* and which perhaps seems be describing *this* commit, but in that case again, I don't understand why is it needed.

Reply:
> I don't understand why this is done. It is committed as part of "Crashed on open cached Crowdin file if logged out", but none of it seems to be related to crashing anything, and surely crashes can be avoided in a simpler way

The same answer as https://github.com/vslavik/poedit/pull/633#discussion_r433356206 . So yep, it's definitely possible to solve it as simple as variant 2 in my above comment if you will decide finally it's most proper solution (i.e. just to silently save locally without bringing `Sign in to Crowdin` UI)

Reply:
> There's also a commit that seems related with a message "Move AttachCloudSync() to GUI code" which does no such thing at all and which perhaps seems be describing this commit, but in that case again, I don't understand why is it needed.

The purpose of this commit is to check (for necessity) and attach (if necessary) cloud sync in each `WriteCatalog()` call doing it in unified way regardless if it's `.xliff`, `.po` and whether opened from local cache or via `Open from Crowdin`. Just forgotten to bring it title/description appropriately when just rebased/refactored/rearranged some another commits where initially cloud sync was attached from some non-GUI code (`SetFileName()` of some of `XLIFFCatalog` classes or so) while those another commits (where it was placed somewhere in non-GUI) successfully gone during that process.  Since earlier it certainly did not work for `Xliff` (since it was not expected/supposed to sync/upload) while cloud sync was attached the only from here  https://github.com/vslavik/poedit/blob/7a0e93fff167ae5aa786d7b74f7e16b78464c88c/src/edframe.cpp#L1102 , i.e. the only from `Open from Crowdin` (so I just don't recall whether it worked for the case when `.po` opened from local cache or especially when downloaded from Crowdin UI in browser and if still worked - then how).

Reply:
Though if you have idea for better place than it is currently is - I would highly appreciate for sharing it here.